assets/src/edit-story/components/panels/textStyle.js

```diff
@@ -0,0 +1,281 @@
+/*
```

Comment:
This seems to be largely very similar to the fonts panel, same comments here as for the font's panel.

Reply:
Indeed. Both are very similar @miina Can you please confirm if we still need that `font` panel? Font's Panel looks like not being in use and redundant.

Reply:
A bit delayed answer: That's correct, the font's panel is redundant, it will be removed within #412

Reply:
Sure! Let me reference this on #255. Thanks for sync up @miina